src/protocols/internal/encode.c

```diff
@@ -57,38 +57,31 @@ static ssize_t internal_encode(fr_dbuff_t *dbuff,
 			       fr_da_stack_t *da_stack, unsigned int depth,
 			       fr_cursor_t *cursor, void *encoder_ctx)
 {
-	uint8_t				*enc_field, *len_field, *value_field, *value_end;
```

Comment:
I really dislike fields being renamed for no purpose.

Reply:
Quite understandable; back they go.

Reply:
Names restored and the result pushed.